### ISSUE TITLE
Improve Project Description

### ISSUE THREAD

**Issue by phlopsi** [corrosion-rs/corrosion#83]
![Screenshot_20210110-005447.png](https://user-images.githubusercontent.com/3685294/104111120-edc10780-52de-11eb-8b29-d5427810bc96.png)

I just stumbled on this project while exploring repositories with the official Github app. The current description is too wordy, leading to a cut-off before the important part. That's unfortunate.

I suggest rewording it to something like "Rust for CMake C/C++ projects". Save the longer sentences for the README.

**Comment by AndrewGaspar** [comment]
@phlopsi Thank you for this feedback!

**Comment by jschwe** [comment]
Closing, since this seems to be resolved.
